packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -212,6 +212,8 @@ void main() {
   });
 
   testUsingContext('can create a skeleton (list/detail) app', () async {
+
+    final bool buildDirAlreadyExists = globals.fs.currentDirectory.childDirectory('build').existsSync();
```

Comment:
Referring to this line here

Reply:
> To get around this, I would first check that there are no build directories in the current working directory and throw an error if the create process created one. If we already had a `build` directory in the current working directory though, the test won't throw a failure if it sees it after the create process
> 
> At first, I didn't have the initial check for the `build` directory in the cwd, but I found that in CI, we would have this directory there already before the test began and I didn't want to assume I could remove it.

If I'm reading into this correctly, this test isn't working with a clean directory. This means one or more other tests are leaking file system state and this should be fixed first. Wdyt?

Reply:
Yeah that's what I'm thinking too. We are using temporary directories for where the files for a new project get created so those are likely clean for each test.  What it seems like is that these `build` directories are getting created at the cwd for each of these tests, which may be at `$FLUTTER_ROOT` so I didn't know if I should assume that it should be clean and not have a `build` directory there as well